src/probnum/linalg/solvers/stopping_criteria/_linear_solver_stopping_criterion.py

```diff
@@ -0,0 +1,31 @@
+"""Base class for linear solver stopping criteria."""
+
+import abc
+
+import probnum  # pylint: disable="unused-import"
+
+
+class LinearSolverStoppingCriterion(abc.ABC):
+    r"""Stopping criterion of a (probabilistic) linear solver.
+
+    Checks whether the linear solver has converged for a given :class:`~probnum.linalg.solvers.ProbabilisticLinearSolverState`.
```

Comment:
What if we don't really run until "convergence"?
How about

```suggestion
    Checks whether quantities tracked by the :class:`~probnum.linalg.solvers.ProbabilisticLinearSolverState` meet a desired terminal condition.
```